consumer-server/test/unit/ping-controller.spec.ts

```diff
@@ -1,36 +1,5 @@
-import { createTestMetricsComponent } from "@well-known-components/metrics"
-import { pingHandler } from "../../src/controllers/handlers/ping-handler"
-import { metricDeclarations } from "../../src/metrics"
-
-describe("ping-controller-unit", () => {
+describe("works", () => {
```

Comment:
are you keeping this file for some reason?

Reply:
To keep the `unit` folder until we have something useful to test there.